multiraft/transport_test.go

```diff
@@ -69,7 +69,10 @@ func (lt *localInterceptableTransport) start() {
 				if !ok {
 					continue
 				}
-				if err := srv.RaftMessage(msg, nil); err != nil {
+				err := srv.RaftMessage(msg, nil)
+				if err == ErrStopped {
```

Comment:
It could, but I prefer to give the condition its own line when it is not the usual `err != nil`.